tests/testthat/test-metadata.R

```diff
@@ -618,7 +618,9 @@ test_that("xportr_*: Domain is kept in between calls", {
   # Divert all messages to tempfile, instead of printing them
   #  note: be aware as this should only be used in tests that don't track
   #        messages
-  withr::local_message_sink(tempfile())
+  if (requireNamespace("withr", quietly = TRUE)) {
```

Comment:
Following CRAN guidelines: https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Suggested-packages

Although {withr} is a required secondary depencency, so (for now) it's very difficult to run tests without having {withr} installed

{vroom} and {tidyselect} import {withr}